test/fixture/power-assert.js

```diff
@@ -0,0 +1,8 @@
+import test from '../../';
+
+test(t => {
+  const a = 'foo';
```

Comment:
Thanks. xo ignores `test/fixture/*` by default and I missed it.
Is there any way to exclude this file from lint ignores?